core/src/main/kotlin/fr/sncf/osrd/api/api_v2/stdcm/STDCMEndpointV2.kt

```diff
@@ -200,6 +222,64 @@ class STDCMEndpointV2(private val infraManager: InfraManager) : Take {
     }
 }
 
+fun buildTemporarySpeedLimitManager(
+    infra: FullInfra,
+    speedLimits: Collection<STDCMTemporarySpeedLimit>
+): TemporarySpeedLimitManager {
+    var outputSpeedLimits: MutableMap<DirTrackChunkId, DistanceRangeMap<SpeedLimitProperty>> =
+        mutableMapOf()
+    for (speedLimit in speedLimits) {
+        for (trackRange in speedLimit.trackRanges) {
+            val trackSection = infra.rawInfra.getTrackSectionFromName(trackRange.trackSection)
+            trackSection?.let {
```

Comment:
```suggestion
            val trackSection = infra.rawInfra.getTrackSectionFromName(trackRange.trackSection) ?: continue
```

There's no need to have this whole extra indentation level

Reply:
Done in https://github.com/OpenRailAssociation/osrd/pull/9837/commits/7a6635a83783a9a0d6ce62dbf50f6c71b1ae1c6d